PsfLauncher/main.cpp

```diff
@@ -304,7 +315,87 @@ void LaunchMonitorInBackground(std::filesystem::path packageRoot, const wchar_t
         }
     }
 }
+
+void LogTelemetryData()
+{
+	auto configRoot = PSFQueryConfigRoot();
+
+	if (auto applications = configRoot->as_object().try_get("applications"))
+	{
+		for (auto& applicationsConfig : applications->as_array())
+		{
+			auto exeStr = applicationsConfig.as_object().try_get("executable")->as_string().wide();
+			auto idStr = applicationsConfig.as_object().try_get("id")->as_string().wide();
+			TraceLoggingWrite(
+				g_Log_ETW_ComponentProvider,
+				"ApplicationsConfigdata",
+				TraceLoggingWideString(exeStr, "applications_executable"),
+				TraceLoggingWideString(idStr, "applications_id"),
+				TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
+				TraceLoggingKeyword(MICROSOFT_KEYWORD_CRITICAL_DATA));
+		}
+	}
+
+	if (auto processes = configRoot->as_object().try_get("processes"))
+	{
+		for (auto& processConfig : processes->as_array())
+		{
+			auto exeStr = processConfig.as_object().get("executable").as_string().wide();
+			TraceLoggingWrite(
+				g_Log_ETW_ComponentProvider,
+				"ProcessesExecutableConfigdata",
+				TraceLoggingWideString(exeStr, "processes_executable"),
+				TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),

```

Comment:
is the AppSessionGuid really required?

Reply:
Not required. I was experimenting with appsession guid to see if I can tie fixup usage with config data and missed removing from this datapoint.

Reply:
I added AppsessionGuid to be able to tie all the events from PSFLauncher